x/protocolpool/keeper/abci.go

```diff
@@ -0,0 +1,37 @@
+package keeper
+
+import (
+	"context"
+	"time"
+
+	"cosmossdk.io/x/protocolpool/types"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (k *Keeper) EndBlocker(ctx context.Context) error {
+	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyEndBlocker)
+
+	logger := k.Logger(ctx).With("module", "x/"+types.ModuleName)
+
+	// Iterate over all continuous fund proposals and perform continuous distribution
+	err := k.ContinuousFund.Walk(ctx, nil, func(key sdk.AccAddress, value types.ContinuousFund) (bool, error) {
+		err := k.continuousDistribution(ctx, value)
+		if err != nil {
+			return false, err
+		}
+
+		logger.Info(
```

Comment:
Let's avoid info logging unless necessary. I don't think it adds anything for a node operator to display this.
Maybe debug level instead?